applications/nrf5340_audio/src/bluetooth/ble_audio_services.c

```diff
@@ -117,6 +127,156 @@ static void vcs_flags_cb_handler(struct bt_vcs *vcs, int err, uint8_t flags)
 	}
 }
 
+#if (CONFIG_AUDIO_DEV == GATEWAY)
+#if (CONFIG_BT_VCS_CLIENT)
+/**
+ * @brief  Callback handler for VCS discover finished
+ *
+ *         This callback handler will be triggered when VCS
+ *         discovery is finished.
+ */
+static void vcs_discover_cb_handler(struct bt_vcs *vcs, int err, uint8_t vocs_count,
+				    uint8_t aics_count)
+{
+	if (err) {
+		LOG_ERR("VCS discover finished callback error: %d", err);
+	} else {
+		LOG_DBG("VCS discover finished");
+	}
+}
+#endif /* (CONFIG_AUDIO_DEV == GATEWAY) */
+#endif /* (CONFIG_BT_VCS_CLIENT) */
+
+#if (CONFIG_BT_MCC)
+/**
+ * @brief  Callback handler for MCS discover finished.
+ *
+ *         This callback handler will be triggered when MCS
+ *         discovery is finished. Used by the client.
+ */
+static void mcc_discover_mcs_cb(struct bt_conn *conn, int err)
+{
+	LOG_DBG("mcc_discover_mcs_cb");
+
+	if (err) {
+		LOG_ERR("Discovery of MCS failed (%d)", err);
+	} else {
+		LOG_DBG("Discovery of MCS finished");
+	}
+}
+
+/**
+ * @brief  Callback handler for sent MCS commands.
+ *
+ *         This callback will be triggered when MCS commands have been sent.
+ *         Only for debugging, used by the client.
+ */
+static void mcc_send_command_cb(struct bt_conn *conn, int err, const struct mpl_cmd *cmd)
+{
+	LOG_DBG("mcc_send_command_cb");
+
+	if (err) {
+		LOG_ERR("Command send failed (%d) - opcode: %u, param: %d", err, cmd->opcode,
+			cmd->param);
+	}
+}
+
+/**
+ * @brief  Callback handler for received notifications.
+ *
+ *         This callback will be triggered when a notification has been received.
+ *         Only for debugging, used by the client.
+ */
+static void mcc_cmd_notification_cb(struct bt_conn *conn, int err, const struct mpl_cmd_ntf *ntf)
+{
+	LOG_DBG("mcc_cmd_ntf_cb");
+
+	if (err) {
+		LOG_ERR("Command notification error (%d) - opcode: %u, result: %u", err,
+			ntf->requested_opcode, ntf->result_code);
+	}
+}
+
+/**
+ * @brief  Callback handler for reading media state.
+ *
+ *         This callback will be triggered when the client has asked to read
+ *         the current state of the media player.
+ */
+static void mcc_read_media_state_cb(struct bt_conn *conn, int err, uint8_t state)
+{
+	LOG_DBG("mcc_read_media_cb");
+
+	if (err) {
+		LOG_ERR("Media State read failed (%d)", err);
+		return;
+	}
+
+	media_player_state = state;
+}
+#endif /* (CONFIG_BT_MCC) */
+
+#if (CONFIG_BT_MCS)
+/**
+ * @brief  Callback handler for received MCS commands.
```

Comment:
Suggest we write out the acronym here.

Reply:
This is not done for VCS either, but both of them are written out in the .h file